### ISSUE TITLE
Custom modules disappear on-click

### ISSUE THREAD

**Issue by luispabon** [Alexays/Waybar#739]
Here's my config: https://github.com/luispabon/sway-dotfiles/tree/master/configs/waybar

I have two custom modules, one for cpu speed and a power button one. As soon as click (right or left) on either, they disappear. Sometimes from all displays, sometimes from the one I clicked on. Sometimes both modules disappear.

 https://youtu.be/ig2MkdemrGg 

The click-action does work - clicking on the cpu speed one does bring a terminal with htop etc.

I can't say when this started happening, but relatively recently.

wlroots 0.10
sway 1.4
Ubuntu 20.04
waybar 4b2e6b54a7fcfa5aae2bbc06f8683bbcf3855849 (1 Jun, latest in master as of writing this)

**Comment by X0rg** [comment]
This issue appeared for me since I build Waybar from the master branch. It may be a regression introduced by fixes for #572.

**Comment by alexarice** [comment]
I'm also getting this problem with a custom spotify module, after running `systemctl --user status waybar` there is the following message:
```
Couldn't map window 0x134f860 as subsurface because its parent is not mapped.
```

**Comment by alexarice** [comment]
I think the message I posted was unrelated, I bisected the issue down to one of

6b32aca094ad108fa4b4a40084bb8fbd6fb118db
6ca4e14b296fa6c1b18e22f28160ba9965ff4bb4
9b9d13ab0d90f16f6d64e7e9dc7394bf80121785

The module didn't show up at all in two of these commits and before the module works fine and after it disappears on click

**Comment by Alexays** [comment]
Fixed with #778

**Comment by luispabon** [comment]
Thank you, I can config that fixed the problem.
